src/partition/repart.c

```diff
@@ -1316,6 +1320,41 @@ static int config_parse_size4096(
         return 0;
 }
 
+static int config_parse_block_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint64_t *blksz = data;
+        int r;
+
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou64(rvalue, blksz);
```

Comment:
hmm, following our usual coding style: please parse this into a local var first, and only after full validation move it into `*blkssz`. we don#t want that by accident an invalid value ends there...

Reply:
Done.